modules/web/js/ballerina/components/editable-text.jsx

```diff
@@ -26,33 +26,46 @@ class EditableText extends React.Component {
             editing: false
         };
 
+        const textBoxPadding = 3;
```

Comment:
I did thought about it. Should we move all the small styling configs there? There are some stuff that definitely should be configurable so should be defined in the config file. But moving every small thing there should make the config unnecessarily huge imo.